app/views/outcomes/show.html.erb

```diff
@@ -67,6 +68,7 @@
         <td><%= assessment.minimum_category%></td>
         <td><%= assessment.target_percentage%></td>
         <td><%= assessment.actual_percentage%></td>
+        <td><%= assessment_type(assessment) %></td>
```

Comment:
I didn't know you could do this! Though I think I might leave the helper for now and revisit if I end up needing this elsewhere.